src/components/Alert/Alert.spec.js

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react'
+import HomePage from '../../pages/HomePage'
+
+it('does not renders the Alert component', () => {
+  const Callback = jest.fn()
```

Comment:
Is there a reason you capitalised `Callback`? I'd write it in all lowercase as it is not a component.